Utilities/aliceHLTwrapper/test/testMessageFormat.cxx

```diff
@@ -17,7 +17,7 @@
 #include "Headers/DataHeader.h"
 #include "Headers/HeartbeatFrame.h"
 
-namespace o2::AliceHLT {
+namespace o2 { namespace AliceHLT {
```

Comment:
is it more readable to open each new namespace on a new line? (not sure what the conventions say)